ingest/expression_files/expression_files.py

```diff
@@ -102,4 +113,46 @@ def set_data_array_gene_expression_values(
             linear_data_id,
             self.study_id,
             self.study_file_id,
-        ).get_data_array()
+        ).get_data_array():
+            yield model
+
+    def load_expression_file(self, gene_docs: List, data_array_documents: List):
+        """
+        Load gene and data_array models into mongoDB
+        """
+        gene_doc_bulk_write_results = None
+        data_array_bulk_write_results = None
+        start_time = datetime.datetime.now()
+
+        # Creating Mongo bulk operations
+        data_array_bulk_operations = list(
+            map(lambda model: InsertOne(model), data_array_documents))
+        gene_model_bulk_operations = list(
+            map(lambda model: InsertOne(model), gene_docs))
+
+        # Try writing data_array_colection
+        try:
+            self.mongo_connection.client['data_arrays'].bulk_write(
+                data_array_bulk_operations,  ordered=False
+            )
+        except BulkWriteError as bwe:
+            print(f"error caused by data docs : {bwe.details}")
+            raise BulkWriteError(f'Error caused by data docs : {bwe.details}')
+
+        except Exception as e:
+            print(f"error caused by data docs : {e}")
+            raise Exception(f'{e}')
+
+        # Try writing gene docs
+        try:
+            gene_doc_bulk_write_results = self.mongo_connection.client[self.COLLECTION_NAME].bulk_write(
+                gene_model_bulk_operations,  ordered=False
+            )
+        except BulkWriteError as bwe:
+            print(f"error caused by gene docs : {bwe.details}")
+            raise BulkWriteError(f'Error caused by data docs : {bwe.details}')
```

Comment:
These seem like they should agree:

```suggestion
            print(f"error caused by gene docs : {bwe.details}")
            raise BulkWriteError(f'Error caused by gene docs : {bwe.details}')
```